Godeps/_workspace/src/github.com/openshift/openshift-sdn/plugins/osdn/subnets.go

```diff
@@ -130,6 +132,17 @@ func (oc *OvsController) SubnetStartNode(mtu uint) error {
 		oc.flowController.AddOFRules(s.NodeIP, s.SubnetCIDR, oc.localIP)
 	}
 
+	pods, err := oc.Registry.GetRunningPods(oc.hostName, kapi.NamespaceAll)
+	if err != nil {
+		return err
+	}
+	for _, p := range pods {
+		err = oc.pluginHooks.UpdatePod(p.Namespace, p.Name, kubetypes.DockerID(p.ContainerID))
+		if err != nil {
+			log.Warningf("Could not update pod %q (%s): %s", p.Name, p.ContainerID, err)
+		}
+	}
```

Comment:
> So can we make oc.flowController.Setup() to tell if update pod is required or not and only perform this op if needed?

Hm... OK, in all the cases where we expect UpdatePod() to do something, we will have destroyed and recreated br0. So we can just move this code and only do it in that case.